components/brave_shields/common/brave_shield_switches.h

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_SHIELDS_COMMON_BRAVE_SHIELD_SWITCHES_H_
+#define BRAVE_COMPONENTS_BRAVE_SHIELDS_COMMON_BRAVE_SHIELD_SWITCHES_H_
+
+namespace brave_shields {
+namespace switches {
+
+// The following command-line switches allow to override defaults of
+// corresponding shields. ATTENTION: using any of this switch persists
+// a given value in content settings of any loaded user profile. Switches
+// behavior mirrors the behavior of toggles on brave://settings/shields.
+//
+// Example: --set-shields-ads-default-for-testing=block
+
+constexpr char kShieldsAdsSetDefaultForTesting[] =
```

Comment:
since their main purpose is internal testing/auditing... I want that external users were able to understand that potential using these switches is their own risk

Reply:
Imho, external users who tamper with command line switches in the first place are a more "advanced" audience. Also, we could maybe have a switch document, similar to [Chrome's](https://peter.sh/experiments/chromium-command-line-switches/#app-shell-refresh-token) where they explicitly state if a switch is for testing purposes without having to incorporate it into the name.

Reply:
@iefremov pls document here:
https://github.com/brave/brave-browser/wiki/Custom-Brave-switches

Reply:
agree with dropping the `for-testing` and `*ForTesting` everywhere